packages/app/src/specs/banners/TrackedBanner.vue

```diff
@@ -8,9 +8,38 @@
   </Alert>
 </template>
 
+<script lang="ts">
+import { CohortConfig, useCohorts } from '@packages/frontend-shared/src/composables/useCohorts'
+
+export type CohortOption = {
+  cohort: string
+  value: string
+}
+
+export const getOptionForCohort = (name: string, options: CohortOption[]) => {
+  const cohortConfig: CohortConfig = {
+    name,
+    cohorts: options.map((option) => option.cohort),
+  }
+
+  const cohortSelected = useCohorts(cohortConfig)
+
+  const emptyOption = { cohort: '', value: '' }
+
+  return computed(() => {
+    if (cohortSelected.value) {
```

Comment:
Since `cohortSelected` is set async after a GraphQL mutation runs, if, for whatever reason, is there any problem if `cohortSelected` is never defined (dashboard down, slow network, some random unexpected error)? Or everything just continues along with no noticeable impact on the user?